frontend/src/App.js

```diff
@@ -22,94 +21,85 @@ function App() {
   const [Sidebar, setSideBar] = useState(null);
   const [MiddleShow, setMiddleShow] = useState("");
 
-  const [all_platforms, setAllPlatforms]= useState([]);
+  const [all_platforms, setAllPlatforms] = useState([]);
   const [Shows, setShows] = useState([]);
   const [Playlists, setPlaylists] = useState([]); // [{name,picture, _id}] --> NEED ROUTE
-  
+
   const [Friends, setFriends] = useState([]);
 
   // _id's of each current active element in each list.
   const [CurrentShow, setCurrentShow] = useState("");
   const [CurrentPlaylist, setCurrentPlaylist] = useState("");
   const [CurrentFriend, setCurrentFriend] = useState("");
-  
+
   const [searchString, setSearch] = useState("");
-  const [currentPage, setCurrentPage] = useState(1);
-  const [count, setCount] = useState(1);  //count of pages
 
-  function listType(title){
-    if(title.includes("Results for") || title == "Recently Added" || title.includes("Shows in")) return "Shows"
-    //else if(List.title == "")
-    return(title);
+  function listType(title) {
+    if (title.includes("Results for") || title == "Recently Added" || title.includes("Shows in") || title.includes("Top Anime")) return "Shows"
```

Comment:
everything else is just prettifying/ indentation... this is what I actually changed